Scene_1/Act_5.txt

```diff
@@ -0,0 +1,14 @@
+Scene 1, Act 5
+
+Enter MRS. PETRUCCI from upstairs
+
+MRS. PETRUCCI: Are your notebook entries finished?
+VICTORIA: Yup
+KATIE: Um.. not quite
+BEN: Yeah, we got a bit distracted programming
+KATIE: Is it ok if we finish at home?
+MRS. PETRUCCI: Fine. But if you don't do it this week, you have to do it first thing next meeting. Got it?
+ALL: Yes
+MRS. PETRUCCI: What's the robot doing? (ROBOT still spinning in circles, now repeatedly saying "Rhyme Know Reason it LIT")
```

Comment:
should be "is lit"

Reply:
You start these comment threads by hovering over the line of code you want to comment on and pressing the plus button that pops up